test-harness/src/main/java/hudson/cli/CLICommandInvoker.java

```diff
@@ -120,6 +124,37 @@ public Result invoke() {
         return new Result(returnCode, out, err);
     }
 
+    private static class GrantPermissions extends AuthorizationStrategy {
```

Comment:
I would like to create an `AuthorizationStrategyBuilder` one day, but it should be more general than this.